boards/arm/rp2040/raspberrypi-pico/src/rp2040_bringup.c

```diff
@@ -102,5 +102,17 @@ int rp2040_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_SENSORS_BMP180
+  /* Try to register BMP180 device in I2C0 */
+
+  ret = board_bmp180_initialize(0);
+
+  if (ret < 0)
+    {
+      syslog(LOG_ERR, "Failed to initialize BMP180 driver: %d\n", ret);
+      return ret;
```

Comment:
This return can be removed.
Otherwise the board will fail to bringup just because of a problem with the sensor.